src/diffusers/utils/peft_utils.py

```diff
@@ -64,9 +64,11 @@ def recurse_remove_peft_layers(model):
             module_replaced = False
 
             if isinstance(module, LoraLayer) and isinstance(module, torch.nn.Linear):
-                new_module = torch.nn.Linear(module.in_features, module.out_features, bias=module.bias is not None).to(
-                    module.weight.device
-                )
+                new_module = torch.nn.Linear(
```

Comment:
Why this change?

Reply:
It's the formatter, when I ran `make style` it made this change

Reply:
Do you prefer if I undo it?